src/LuaTransformer.ts

```diff
@@ -315,14 +315,23 @@ export class LuaTransformer {
         const moduleSpecifier = statement.moduleSpecifier as ts.StringLiteral;
         const importPath = moduleSpecifier.text.replace(new RegExp("\"", "g"), "");
 
-        const requireCall = this.createModuleRequire(statement.moduleSpecifier as ts.StringLiteral);
-
         if (!statement.importClause) {
+            const requireCall = this.createModuleRequire(statement.moduleSpecifier as ts.StringLiteral);
             result.push(tstl.createExpressionStatement(requireCall));
             return result;
         }
 
         const imports = statement.importClause.namedBindings;
+        const type = this.checker.getTypeAtLocation(imports);
+        let requireCall: tstl.CallExpression;
+        if (type && this.isAmbientModuleDeclaration(type)) {
+            requireCall = tstl.createCallExpression(
+                tstl.createIdentifier("require"),
```

Comment:
Might be nicer to delegate this logic to `createModuleRequire` by adding an optional `resolveModule: bool = true`  parameter.